sdk/programmableconnectivity/azure-programmableconnectivity/dev_requirements.txt

```diff
@@ -0,0 +1,5 @@
+-e ../../../tools/azure-devtools
```

Comment:
remove the azure-devtools line, we deleted this from the repo since it's not needed and it's causing the CI to fail since it can't find it

Reply:
will remove. However, should something be raised about the fact that it's being put in here from the autogenerate step?

Reply:
Good point, let me double-check on that.